src/BenchmarkDotNet.Core/Running/ClassicBenchmarkConverter.cs

```diff
@@ -1,9 +1,11 @@
 ﻿#if CLASSIC
 using System;
 using System.CodeDom.Compiler;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using System.Net;
+using System.Runtime.Remoting.Activation;
```

Comment:
what is the purpose of having the dependency to `System.Runtime.Remoting.Activation`?

Reply:
Fixed